src/test/java/org/opensearch/neuralsearch/query/HybridQueryAggregationsIT.java

```diff
@@ -100,43 +100,43 @@ protected boolean preserveClusterUponCompletion() {
 
     @SneakyThrows
     public void testPipelineAggs_whenConcurrentSearchEnabled_thenSuccessful() {
```

Comment:
Do not review this class as it already merged in main. The change is showing here because we have cherry picked the commit.